test/Invio.QueryProvider.Test.CSharp/QueryableTestBase.cs

```diff
@@ -489,24 +530,179 @@ public abstract class QueryableTestBase {
+
+            AssertCustomerIdSet(
+                CustomerIdsWithCompanyNamesContainingQULowerCase,
+                results
+            );
+        }
+
+        [Theory]
+        [InlineData("ANTON", new[] { "ANTON" })]
+        [InlineData("anton", new String[0])]
+        public virtual void Where_String_EqualsOperator_DefaultsToCaseSensitive(
+            String query,
+            String[] expectedResult) {
+
+            var results = this.Customers.Where(c => c.CustomerId == query).ToList();
+
+            AssertCustomerIdSet(expectedResult, results);
+        }
+
+        [Theory]
+        [InlineData("ANTON", new[] { "ANTON" })]
+        [InlineData("anton", new String[0])]
+        public virtual void Where_String_EqualsFunction_DefaultsToCaseSensitive(
+            String query,
+            String[] expectedResult) {
+
+            var results = this.Customers.Where(c => c.CustomerId.Equals(query)).ToList();
+
+            AssertCustomerIdSet(expectedResult, results);
+        }
+
+        [Theory]
+        [InlineData("ANTON", new[] { "ANTON" })]
+        [InlineData("anton", new String[0])]
+        public virtual void Where_String_StaticEqualsFunction_DefaultsToCaseSensitive(
+            String query,
+            String[] expectedResult) {
+
+            var results = this.Customers.Where(c => String.Equals(c.CustomerId, query)).ToList();
+
+            AssertCustomerIdSet(expectedResult, results);
+        }
+
+        [Theory]
+        [InlineData("ANTON", StringComparison.Ordinal, new[] { "ANTON" })]
+        [InlineData("ANTON", StringComparison.InvariantCulture, new[] { "ANTON" })]
+        [InlineData("anton", StringComparison.Ordinal, new String[0])]
+        [InlineData("anton", StringComparison.InvariantCulture, new String[0])]
+        [InlineData("anton", StringComparison.OrdinalIgnoreCase, new[] { "ANTON" })]
+        [InlineData("anton", StringComparison.InvariantCultureIgnoreCase, new[] { "ANTON" })]
+        public virtual void Where_String_EqualsFunction_CaseSensitivity(
+            String query,
+            StringComparison comparison,
+            String[] expectedResult) {
+
+            var results =
+                this.Customers.Where(c => c.CustomerId.Equals(query, comparison)).ToList();
+
+            AssertCustomerIdSet(expectedResult, results);
+        }
+
+        [Theory]
+        [InlineData("ANTON", StringComparison.Ordinal, new[] { "ANTON" })]
+        [InlineData("ANTON", StringComparison.InvariantCulture, new[] { "ANTON" })]
+        [InlineData("anton", StringComparison.Ordinal, new String[0])]
+        [InlineData("anton", StringComparison.InvariantCulture, new String[0])]
+        [InlineData("anton", StringComparison.OrdinalIgnoreCase, new[] { "ANTON" })]
+        [InlineData("anton", StringComparison.InvariantCultureIgnoreCase, new[] { "ANTON" })]
+        public virtual void Where_String_StaticEqualsFunction_CaseSensitivity(
```

Comment:
As both `Where_String_StaticEqualsFunction_CaseSensitivity` and `Where_String_EqualsFunction_CaseSensitivity` have the same test cases, they can be consolidated into a single block of member data, `[MemberData(nameof(Where_StringEquals_CaseSensitivity))]`